tests/unit/string_util_test.cpp

```diff
@@ -0,0 +1,57 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+
+#include <gtest/gtest.h>
+
+using namespace testing;
+
+extern "C" {
+   #include <tilck/common/string_util.h>
+}
+
+TEST(strstr, basic)
+{
+   const char haystack[] = "haystack", needle[] = "needle", \
```

Comment:
I forgot to mention this cosmetic detail, but this style is really UGLY to me. Don't need to fix it as I'm preparing a change now and will fix it too. Just, for the next time, do something like:

```C
const char *haystack = "haystack";
const char *needle = "needle";
const char *hay = "hay";
const char *empty = "";
```

Better to split the declarations across multiple lines and try NEVER to use the backslash. In this case it was also **totally pointless**.

Reply:
Ok, perfect. Sorry for the inconvenience